### PR TITLE
Let pointmap send a query for each zoom/drag.

### PR DESCRIPTION
**Pull request by DeyuChen**
Let pointmap send a query for each zoom/drag to update the points and the time series. The query should use the geoIds filter of the current viewing area.

### CODE REVIEW

**Comment by chenlica** [comment]
@waans11 and @baiqiushi can review this PR.

File: examples/twittermap/web/public/javascripts/map/controllers.js [hunk 1]
```diff
@@ -554,6 +554,9 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
             if ($scope.polygons.statePolygons) {
               $scope.map.removeLayer($scope.polygons.statePolygons);
             }
+            if ($scope.polygons.statePolygons) {
+              $scope.map.removeLayer($scope.polygons.statePolygons);
+            }
```
Comment (waans11) [inline examples/twittermap/web/public/javascripts/map/controllers.js]: Why is this needed? Seems a duplicate code?

File: examples/twittermap/web/public/javascripts/map/controllers.js [hunk 2]
```diff
@@ -564,6 +567,9 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
             $scope.map.addLayer($scope.polygons.countyPolygons);
           } else if ($scope.status.zoomLevel <= 5) {
             resetGeoInfo("state");
+            if ($scope.polygons.statePolygons) {
+              $scope.map.removeLayer($scope.polygons.statePolygons);
+            }
```
Comment (waans11) [inline examples/twittermap/web/public/javascripts/map/controllers.js]: State Polygons (zoom level <=5) should be removed?

File: examples/twittermap/web/public/javascripts/map/controllers.js (hunk 1 above)
Comment (DeyuChen) [reply examples/twittermap/web/public/javascripts/map/controllers.js]: Sorry I committed from a wrong folder.

File: examples/twittermap/web/public/javascripts/map/controllers.js [hunk 3]
```diff
@@ -609,8 +615,11 @@ angular.module('cloudberry.map', ['leaflet-directive', 'cloudberry.common','clou
         }
         if ($scope.status.logicLevel === 'city') {
             loadCityJsonByBound(onEachFeature, false);
+        } else {
+          resetGeoIds($scope.bounds, geoData, $scope.status.logicLevel + "ID");
```
Comment (waans11) [inline examples/twittermap/web/public/javascripts/map/controllers.js]: resetGeoIds are required for both state and county level, too. I think. Check the original line 613.

Comment (baiqiushi) [reply examples/twittermap/web/public/javascripts/map/controllers.js]: I think this "else" is actually for both "state" and "county" level.
